### PR TITLE
Piezol zad4

### LINKED ISSUE #4: Zadanie 4 (Map)
Napisz klasę Map, która będzie posiadała `std::vector<Island> `przechowujący wszystkie wyspy na mapie, oraz zmienną `Island* currentPosition_` określającą aktualną pozycję gracza na mapie.

### PR DESCRIPTION
**Pull request by piezol**

### CODE REVIEW

File: shm/inc/map.hpp [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <vector>
+
+#include "island.hpp"
+
+class Map {
+private:
+    std::vector<Island> island_;
```
Comment (kontrybutor) [inline shm/inc/map.hpp]:
```suggestion
    std::vector<Island> islands_;
```
I think it's more meaningful

File: shm/inc/map.hpp [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <vector>
+
+#include "island.hpp"
```
Comment (kontrybutor) [inline shm/inc/map.hpp:5]:
```suggestion
#include "island.hpp"

#include <vector>
```
The other files have include order as above. First coresponding header, then local header, then system header. I think it's good idea to stick to one convention. But it's up to you.

Comment (piezol) [reply shm/inc/map.hpp:5]: It's the other way around.
See https://coders.school/edu/courses/powtorka-1-qa/lesson/kolejnosc-include/.

I have my VSC set in a way, that would revert the order of headers automatically if i applied your suggestion

File: shm/inc/map.hpp (hunk 1 above)
Comment (piezol) [reply shm/inc/map.hpp]: Correct, thanks

**Review by kontrybutor — APPROVED** [review]

**Review by arafalski — APPROVED** [review]

File: shm/inc/map.hpp (hunk 2 above)
Comment (piezol) [reply shm/inc/map.hpp:5]: (it's enabled by default by the chromium style)

**Pull request merged**